### DISCUSSION TITLE
Project creation

### DISCUSSION THREAD

**Discussion by Staffilaffi** [opengisch/QField#5443] [Q&A]
Hi, I created a project in QGIS to insert it into QField via cloud. My project is simple: Google Maps and a single point layer. When I tried to create a project for the cloud, the project choice is not the recommended one, which is the current project, but only the underlying option to create a new project. Why do I have this problem? Thanks everyone!
![Immagine 2024-07-15 213722](https://github.com/user-attachments/assets/56873c38-01ca-4f9a-a20b-80ba870c555e)

**Comment by SeqLaz** [comment]
Hello @Staffilaffi , could you check if the project has already been converted to a QFieldCloud project? To do this, enter the properties of the project by clicking on Project > Properties or by pressing CTRL + Shift + P, and verify where the location of the project is. Additionally, check if the project has the title "QFieldCloud".

![image](https://github.com/user-attachments/assets/7b8af458-787a-41d9-88c5-fd41b2487f5a)

**Reply by Staffilaffi** [reply]
Thanks, I try
